### DISCUSSION TITLE
Add a checkbox in the product sheet as a reminder if the product needs to be restocked

### DISCUSSION THREAD

**Discussion by ethereastore** [PrestaShop/PrestaShop#40145] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+) and found no previous suggestion of this feature.

### Is your feature request related to a problem?

I have a site in prestashop ethereastore.it. Having many products I find myself in difficulty when I make restocking in catalog-> stock because many products do not have to be restocked even when the quantity is zero.

I do the restocking in catalog-> stock I order by quantity and restock the products with quantity 0/1. for various reasons some products must no longer be restocked. Having more than a thousand products it becomes difficult to remember. for now when a product does not need to be restocked I add a * at the beginning of the product name.

### Describe the solution you'd like

I think a checkbox in the product sheet is useful as a reminder if the product should not be restocked in the future and be visible in catalog-> stock.
 I think it's a very useful feature for someone with a lot of products. Thank you

### Do you plan to work on this feature?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by ethereastore** [comment]
thanks but that's not what i need.
I do the restocking in catalog-> stock I order by quantity and restock the products with quantity 0/1. for various reasons some products must no longer be restocked. Having more than a thousand products it becomes difficult to remember. for now when a product does not need to be restocked I add a * at the beginning of the product name. I think a checkbox in the product sheet is useful as a reminder if the product should not be restocked in the future and be visible in catalog-> stock.

**Comment by ethereastore** [comment]
thank you!! I believe it is a convenient feature for many. Sorry about my english, i'm using google translator :-)

Cordiali Saluti
Etherea Store Riccione

**Comment by hibatallahAouadni** [comment]
Hello @ethereastore 

Thank you for your suggestion. The Product Team will take it into consideration for future developments. 
Please be aware that there is no guarantee that this feature will be developed anytime soon. 

Thans @buggyzap for your contribution and we're waiting for your PR :rocket: 

Thank you

**Comment by ethereastore** [comment]
Thank you. the demonstration video is perfect!

Ps. Sapevo che eri italiano perché dalla tua foto del profilo GitHub lo ho riconosciuta ad alcuni video su YouTube che ti riguardavano parlando delle novità di prestashop e dei moduli prestashop essentials (con i moduli Marketing with Google e Ps Facebook ho trovato molti problemi ma questa è un’altra storia). Ho provato di scrivere in inglese perché tutti la potessero capire. E’ la prima volta che partecipo alla pool request e non ho capito bene come funziona. Ti ringrazio per il tuo contributo che dai per prestashop, trovo questa CMS meraviglioso. Prima usavo Shopify e ora sono contentissimo di essere passato prestashop 😊

**Comment by hibatallahAouadni** [comment]
@buggyzap @ethereastore Please use English!! English is the norme on GitHub :wink:
